src/shared/components/challenge-detail/Header/TabSelector/index.jsx

```diff
@@ -77,11 +77,16 @@ export default function ChallengeViewSelector(props) {
           styleName={getSelectorStyle(selectedView, DETAIL_TABS.DETAILS, trackLower)}
         >DETAILS
         </a>
-        <a
-          onClick={(e) => { handleSelectorClicked(e, DETAIL_TABS.REGISTRANTS); }}
-          styleName={getSelectorStyle(selectedView, DETAIL_TABS.REGISTRANTS, trackLower)}
-        >REGISTRANTS {numRegistrants ? `(${numRegistrants})` : ''}
-        </a>
+        {
```

Comment:
For such conditional blocks we prefer to use `?:` operator, also no need to compare with zero then; please update to:
```
{
  numRegistrants ? (
    <a
      ...
    >...</a>
  ) : null
}
```